tests/run-separate-compilation/f-interpolation-1/FQuote_1.scala

```diff
@@ -0,0 +1,58 @@
+import scala.quoted._
+import scala.tasty.Reflection
+
+import scala.language.implicitConversions
+
+object FQuote {
+
+  implicit class SCOps(ctx: StringContext) {
```

Comment:
Just to confirm: I remember some time ago, we need to use `=> StringContext`. I guess with `underlyingArgument` this is no longer required.